draft-ietf-quic-transport.md

```diff
@@ -926,8 +926,11 @@ server continues below.
 
 If the packet is an Initial packet fully conforming with the
 specification, the server proceeds with the handshake ({{handshake}}).
-This commits the server to the version that the client
-selected.
+This commits the server to the version that the client selected.
+
+If the server currently doesn't accept any new connections, it SHOULD send a
+SERVER_BUSY error code. A client that receives this error code SHOULD NOT
+immediately try to reconnect to this server.
```

Comment:
"send an error code" isn't the same terminology we've used elsewhere.  I'd use something like "If a server isn't currently accepting any new connections, it SHOULD send a Handshake packet containing a CONNECTION_CLOSE frame with error code SERVER_BUSY."

Reply:
Thanks for the suggestion. That sounds better.